lib/evaluation/evaluationClient.ts

```diff
@@ -69,12 +64,20 @@ export class EvaluationClient {
     this._did = did;
   }
 
-  public get verifiablePresentation(): VerifiablePresentation {
-    return this._verifiablePresentation;
+  public get presentationSubmission(): PresentationSubmission {
+    return this._presentationSubmission as PresentationSubmission;
   }
 
-  public set verifiablePresentation(verifiablePresentation: VerifiablePresentation) {
-    this._verifiablePresentation = verifiablePresentation;
+  public set presentationSubmission(presentationSubmission: Partial<PresentationSubmission>) {
+    this._presentationSubmission = presentationSubmission;
+  }
+
+  public get verifiableCredential(): VerifiableCredential[] {
+    return this._verifiableCredential as VerifiableCredential[];
```

Comment:
It's needed because it is a partial

Reply:
The purpose of the comment was to avoid using as if possible. If current way is the only way to do it then please comment here. We can close the comment.